app/main/ipc-helpers.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * This file contains the functions which make use of Main process APIs with already functioning code in Renderer process.
+ */
+import {Notification, nativeImage} from 'electron';
+
+import fetch from 'node-fetch';
```

Comment:
`node-fetch` doesn’t use the Electron network stack, so this won’t work with custom certificates, proxies, etc. We should use `electron.net` like everything else.